docs/Windows.md

```diff
@@ -30,9 +30,30 @@ We recommend using packages from [conda-forge][2].
 Launch cmd.exe and run following to bootstrap a build environment:
 
 ```shell
-conda create -n parquet-dev cmake git boost-cpp curl zlib snappy -c conda-forge
+conda create -n parquet-dev cmake git boost-cpp curl zlib snappy arrow-cpp brotli thrift-cpp -c conda-forge
```

Comment:
Can you remove arrow-cpp from here since it will change faster than what's in conda-forge?